JASP-Engine/JASP/R/descriptives.R

```diff
@@ -975,6 +993,68 @@ Descriptives <- function(jaspResults, dataset, options) {
   return(thePlot)
 }
 
+.descriptivesIntervalPlot <- function(dataset, options,  variable) {
+
+  # Define custom y axis function
+  base_breaks_y <- function(x) {
+    b <- pretty(x)
+    d <- data.frame(x=-Inf, xend=-Inf, y=min(b), yend=max(b))
+    list(ggplot2::geom_segment(data=d, ggplot2::aes(x=x, y=y, xend=xend, yend=yend), size = 0.75, inherit.aes=FALSE), ggplot2::scale_y_continuous(breaks=b))
+  }
+
+  thePlot <- createJaspPlot(title=variable, width=options$plotWidth, height=options$plotHeight)
+
+  errorMessage <- .descriptivesCheckPlotErrors(dataset, variable, obsAmount = "< 1")
+  if (!is.null(errorMessage)) {
+    thePlot$setError(gettextf("Plotting not possible: %s", errorMessage))
+  }
+  else {
+    # we need to know which index in y is related to which index in the actual data, so we should not forget the NAs somehow, lets make a list of indices.
+    yWithNA         <- dataset[[.v(variable)]]
+    y               <- na.omit(dataset[[.v(variable)]])
+    yIndexToActual  <- y
+    yWithNAIndex    <- 1
+    yNoNAIndex      <- 1
+    while(yWithNAIndex <= length(yWithNA)) {
+
+      if(!is.na(yWithNA[[yWithNAIndex]])) {
+
+        yIndexToActual[[yNoNAIndex]] <- row.names(dataset)[[yWithNAIndex]]
+        yNoNAIndex                   <- yNoNAIndex + 1
+      }
+
+      yWithNAIndex <- yWithNAIndex + 1
+    }
+
+    if (is.null(dataset[[.v(options$splitby)]])){
+      group     <- factor(rep("",length(y)))
+      xlab      <- "Total"
+
+    } else {
+      group     <- as.factor(dataset[[.v(options$splitby)]])[!is.na(dataset[[.v(variable)]])]
+      xlab      <- options$splitby
+
+    }
+
+    plotDat <- data.frame(group = group, y = y)
+    row.names(plotDat) <- yIndexToActual
```

Comment:
```suggestion
    rownames(plotDat) <- yIndexToActual
```
I know the result is the same but we use `rownames` almost everywhere in the codebase.